src/layouts/partials/changes/main.html

```diff
@@ -30,7 +30,7 @@ <h3>By Category</h3>
   </div>
   <div class="col-sm-12 col-md-8 base-content">
 
-    {{ if and (ne .Kind "page") (gt .Paginator.TotalPages 1) }}
```

Comment:
This was a nasty bug where a new provider with few entries were not displayed properly